src/appkit.cs

```diff
@@ -11018,20 +11230,45 @@ partial interface NSPasteboard // NSPasteboard does _not_ implement NSPasteboard
 		// Pasteboard names: for NSPasteboard.FromName()
 
 		[Field ("NSGeneralPboard")]
+		[Availability (Deprecated = Platform.Mac_10_13, Message = "Use 'NSPasteboardNameGeneral' instead")]
 		NSString NSGeneralPasteboardName { get; }
 
 		[Field ("NSFontPboard")]
+		[Availability (Deprecated = Platform.Mac_10_13, Message = "Use 'NSPasteboardNameFont' instead")]
 		NSString NSFontPasteboardName { get; }
 
 		[Field ("NSRulerPboard")]
+		[Availability (Deprecated = Platform.Mac_10_13, Message = "Use 'NSPasteboardNameRuler' instead")]
 		NSString NSRulerPasteboardName { get; }
 
 		[Field ("NSFindPboard")]
+		[Availability (Deprecated = Platform.Mac_10_13, Message = "Use 'NSPasteboardNameFind' instead")]
 		NSString NSFindPasteboardName { get; }
 
 		[Field ("NSDragPboard")]
+		[Availability (Deprecated = Platform.Mac_10_13, Message = "Use 'NSPasteboardNameDrag' instead")]
 		NSString NSDragPasteboardName { get; }
 
+		[Mac (10, 13)]
+		[Field ("NSPasteboardNameGeneral")]
+		NSString NSPasteboardNameGeneral { get; }
```

Comment:
No, but FromName accepts arbitrary names (https://developer.apple.com/documentation/appkit/nspasteboard/1531026-pasteboardwithname?language=objc) so I didn't think I could create an override that accepted some smart enum without obscuring that.